packages/central-server/app/sync/CentralSyncManager.js

```diff
@@ -359,8 +365,11 @@ export class CentralSyncManager {
 
       await this.waitForPendingEdits(tick);
 
+      const { minSourceTick, maxSourceTick } =
+        (await getLookupSourceTickRange(this.store, since, tick)) || {};
```

Comment:
For some reason I really don't love a bracketed await. Is there a time when `getLookupSourceTickRange` returns something other than an object? Can we fix it there instead of having this ternary?